### PR TITLE
[BugFix] Support initialize llumlet in manager

### PR DESCRIPTION
**Pull request by s5u13b**
When the migration backend is gloo, during scale_up/scale_down, llumnix needs to re-initialize the migration communication process group. However, Ray requires that all the processes that attempt to initialize the process group share the same driver process to ensure that the job_id is consistent. This means that all the llumlets should be initialized in the same process in llumnix. Previously, llumnix initialize llumlet in the entrypoint process, and we can have multiple entrypoints in llumnix, so the driver process of llumlets can be different. Therefore, we support initializing llumlet in the manager as another option in llumnix to ensure that all the llumlets can share the same driver process(manager).

### CODE REVIEW

File: llumnix/backends/utils.py [hunk 1]
```diff
@@ -55,8 +56,9 @@ def initialize_cluster(
             "Ray is not installed. Please install Ray to use distributed "
             "serving.")
     # Connect to a ray cluster.
-    ray.init(address=ray_address, ignore_reinit_error=True)
+    ray.init(address=ray_address, ignore_reinit_error=True, namespace='llumnix')
```
Comment (KuilongCui) [inline llumnix/backends/utils.py:60]: get_placementgroup maybe better for this method.
mv ray.init to ap_server after launch_ray_cluster

File: llumnix/entrypoints/llumnix_utils.py [hunk 2]
```diff
@@ -158,23 +159,28 @@ def init_request_output_queue() -> RayQueue:
...
-                            engine_args: AsyncEngineArgs) -> Tuple[LLMEngineManager, List[Llumlet], RayQueue]:
+                            engine_args,
+                            node_id: str) -> Tuple[LLMEngineManager, List[Llumlet], RayQueue]:
     assert engine_args.engine_use_ray and engine_args.worker_use_ray, \
             ("In Llumnix, engine and worker must be ray actor in orther to run step and migrate concurrently.")
-
     engine_manager = init_manager(engine_manager_args)
-    logger.info("Init LLMEngineManager done")
-    instance_ids, llumlets = init_llumlets(engine_manager_args, engine_args)
+    if not engine_manager_args.init_instance_in_manager:
```
Comment (KuilongCui) [inline llumnix/entrypoints/llumnix_utils.py:167]: maybe we can add a parameters checker when parse arguments in api_server.
don't do this in lazy model

Comment (s5u13b) [reply llumnix/entrypoints/llumnix_utils.py:167]: Good suggestion, we can create a seperate pr to support parameters checker in llumnix.

File: llumnix/backends/utils.py (hunk 1 above)
Comment (s5u13b) [reply llumnix/backends/utils.py:60]: Get it.

File: llumnix/backends/utils.py [hunk 3]
```diff
@@ -35,6 +35,7 @@ def init_backend_engine(instance_id: str, backend_type: BackendType, *args, **kw
 def initialize_cluster(
     world_size: int = 1,
     ray_address: Optional[str] = None,
+    detached: bool = False,
```
[the remark this answers is not in the document]
Comment (zhypku) [reply llumnix/backends/utils.py:38]: sure

File: llumnix/entrypoints/vllm/api_server.py [hunk 4]
```diff
@@ -235,7 +235,7 @@ async def is_ready():
     engine_manager_args = EngineManagerArgs.from_cli_args(args)
     engine_args = AsyncEngineArgs.from_cli_args(args)
 
-    logger.info("engine_args: {}".format(engine_args))
+    print("engine_args: {}".format(engine_args))
```
[the remark this answers is not in the document]
Comment (s5u13b) [reply llumnix/entrypoints/vllm/api_server.py:238]: logger cannot output here and also in api server process for some unknown reason.

File: docs/Arguments.md [hunk 5]
```diff
@@ -35,9 +36,12 @@ usage: -m llumnix.entrypoints.vllm.api_server [-h]
             [--max-stages MAX_STAGES]
 ```
 
-`--fixed-node-init`
+`--fixed-node-init-instance`
 - Fix the placement of instance to current node.
 
+`--init-instance-in-manager`
```
[the remark this answers is not in the document]
Comment (s5u13b) [reply docs/Arguments.md:42]: Get it. I will change it to 'init-instance-by-manager'.

**Review by ZeldaHuang — APPROVED** [review]

**Review by KuilongCui — APPROVED** [review]

**Review by KuilongCui — APPROVED** [review]

**Pull request merged**